src/components/Navigator.js

```diff
@@ -0,0 +1,88 @@
+import React, { useEffect } from 'react';
+import {
+  Route,
+  Routes,
+  Link,
+  useLocation,
+} from 'react-router-dom';
+import { useSelector, useDispatch } from 'react-redux';
+import { getRockets } from '../redux/rockets/rockets';
+import { getMissions } from '../redux/missions/missions';
+import Profile from './Profile';
+import Rockets from './Rockets';
+import Missions from './Missions';
+import './Navigator.css';
+
+const toggleActive = (e) => {
+  e.target.parentNode.parentNode.childNodes.forEach((child) => {
+    if (child.classList.contains('active')) {
+      const child2 = child;
+      child2.className = 'inactive';
+    }
+  });
+  e.target.parentNode.className = 'active';
+};
+
+const Navigator = () => {
+  const rockets = useSelector((state) => state.rockets);
+  const missions = useSelector((state) => state.missions);
+  const location = useLocation();
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    const profile = document.getElementById('profile');
+    const missions = document.getElementById('missions');
+    const rockets = document.getElementById('rockets');
+
+    if (profile && missions && rockets) {
+      switch (location.pathname) {
+        case '/profile':
+          profile.className = 'active';
+          missions.className = 'inactive';
+          rockets.className = 'inactive';
+          break;
+        case '/mission':
+          missions.className = 'active';
+          rockets.className = 'inactive';
+          profile.className = 'inactive';
+          break;
+        default:
+          rockets.className = 'active';
+          profile.className = 'inactive';
+          missions.className = 'inactive';
+      }
+    }
+    dispatch(getRockets());
+    dispatch(getMissions());
+  }, []);
+
+  return (
+    <div className="main">
+      <div className="header">
+        <img src="https://user-images.githubusercontent.com/20567503/161595022-dd594e3e-cadc-465c-8cbb-38d0467b2da5.png" alt="Website logo" id="logo" />
+        <h1 className="title">Space Travellers&apos; Hub</h1>
+        <ul className="menu-items">
+          <li id="rockets">
+            <Link to="/" onClick={toggleActive} className="link">Rockets</Link>
+          </li>
+          <li id="missions">
+            <Link to="/mission" onClick={toggleActive} className="link">Missions</Link>
+          </li>
+          <li>
+            <div className="vertical-line" />
+          </li>
+          <li id="profile">
+            <Link to="/profile" onClick={toggleActive} className="link">My Profile</Link>
```

Comment:
- [ ] Nice use of the `Link` component for the navigation links, however, it is required to use the `NavLink` component for the navigation links instead of the `Link` component. The difference between them is that the Link component is used to navigate the different routes on the site. But NavLink is used to add the style attributes to the active route. Check this really useful [resources](https://dev.to/yunweneric/implementing-react-routes-part-2-link-vs-navlink-5d6e) to learn more about the difference between Link and NavLink. So instead of having the function `toggleActive`, please make use of NavLink so that you can learn how to do that effortlessly and follow requirements too😄.

Reply:
Thanks for the review, I will make sure to use NavLink instead.

Reply:
Thanks again @julie-ify  this made everything so easy and I was complicating it using Link instead haha.